--- contrib/a52dec/module.defs.orig	2010-12-29 22:55:08.403590863 +0100
+++ contrib/a52dec/module.defs	2010-12-29 22:52:46.521442273 +0100
@@ -5,4 +5,4 @@
 A52DEC.EXTRACT.tarbase = a52dec
 A52DEC.INSTALL.strip = liba52.a
 
-A52DEC.CONFIGURE.bootstrap = rm -fr aclocal.m4 autom4te.cache; autoreconf -fiv;
+A52DEC.CONFIGURE.bootstrap = rm -fr aclocal.m4 autom4te.cache; aclocal-1.11; autoreconf-2.68 -fiv;
--- gtk/configure.ac.orig	2011-02-26 09:44:10.298233283 +0100
+++ gtk/configure.ac	2011-02-26 09:45:11.867793425 +0100
@@ -95,9 +95,9 @@
     ;;
   *)
     if test "x$gst_disable" = "xyes" ; then
-	    GHB_PACKAGES="gtk+-2.0 >= 2.10 gthread-2.0 gio-2.0 libnotify gudev-1.0"
+	    GHB_PACKAGES="gtk+-2.0 >= 2.10 gthread-2.0 gio-2.0 libnotify gudev-1.0 dbus-glib-1"
     else
-	    GHB_PACKAGES="gtk+-2.0 >= 2.10 gthread-2.0 gstreamer-0.10 gstreamer-interfaces-0.10 gstreamer-video-0.10 gstreamer-pbutils-0.10 gio-2.0 libnotify gudev-1.0"
+	    GHB_PACKAGES="gtk+-2.0 >= 2.10 gthread-2.0 gstreamer-0.10 gstreamer-interfaces-0.10 gstreamer-video-0.10 gstreamer-pbutils-0.10 gio-2.0 libnotify gudev-1.0 dbus-glib-1"
     fi
 	mingw_flag=no
     ;;
