diff -Naur git-1.6.4.4.orig/Makefile git-1.6.4.4/Makefile
--- git-1.6.4.4.orig/Makefile	2009-09-16 23:11:52 +0000
+++ git-1.6.4.4/Makefile	2009-10-25 11:37:18 +0000
@@ -654,6 +654,12 @@
 # because maintaining the nesting to match is a pain.  If
 # we had "elif" things would have been much nicer...
 
+#
+# For uclib system, there is NO_NSEC
+#
+
+NO_NSEC = YesPlease
+
 ifeq ($(uname_S),Linux)
 	NO_STRLCPY = YesPlease
 	NO_MKSTEMPS = YesPlease
